decoder/stats.go

```diff
@@ -246,20 +247,24 @@ func updatePokemonStats(old *Pokemon, new *Pokemon, areaNames []geo.AreaName) {
 
 			if old == nil || old.PokemonId != new.PokemonId { // pokemon is new or type has changed
 				countStats.count[new.PokemonId]++
+				external.PokemonCountNew.WithLabelValues(area.String()).Inc()
+				if new.ExpireTimestampVerified {
+					external.UpdateVerifiedTtl(area, new.SeenType, new.ExpireTimestamp)
+				}
 			}
 			if new.Cp.Valid {
 				countStats.ivCount[new.PokemonId]++
-				if new.Shiny.ValueOrZero() {
-					countStats.shiny[new.PokemonId]++
```

Comment:
@jfberry @lenisko so we decided to remove the shiny stats completely? we don't write it to DB anymore?